<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdaterDialog</class>
 <widget class="QDialog" name="UpdaterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset theme="system-software-update">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="dialogLayout">
   <item row="5" column="1">
    <widget class="QPushButton" name="installButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Exit the program and run the installer</string>
     </property>
     <property name="text">
      <string>Install</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="downloadButton">
     <property name="toolTip">
      <string>Download the installer to the Downloads folder</string>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A new version of Crow Translate is available! Updates add functionality and improve the stability of the application. Most often.&lt;br/&gt;You can also download the release manually from this &lt;a href=&quot;https://github.com/crow-translate/crow-translate/releases&quot;&gt;link&lt;/a&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QProgressBar" name="downloadBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="updateLaterButton">
     <property name="toolTip">
      <string>Close this window</string>
     </property>
     <property name="text">
      <string>Update later</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="updateStatusLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="cancelDownloadButton">
     <property name="toolTip">
      <string>Cancel download</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTextEdit" name="changelogTextEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="currentVersionTitleLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Current version:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="currentVersionLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="availableVersionTitleLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Available version:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="availableVersionLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelDownloadButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdaterDialog</receiver>
   <slot>cancelDownload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdaterDialog</receiver>
   <slot>download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>updateLaterButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdaterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>582</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>installButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdaterDialog</receiver>
   <slot>install()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>install()</slot>
  <slot>cancelDownload()</slot>
  <slot>download()</slot>
 </slots>
</ui>
